src/formatters/checkstyleFormatter.ts

```diff
@@ -3,7 +3,7 @@ import {RuleFailure} from "../language/rule/rule";
 
 export class Formatter extends AbstractFormatter {
     public format(failures: RuleFailure[]): string {
-        let output = '<?xml version="1.0" encoding="utf-8"?><checkstyle version="4.3">';
+        let output = "<?xml version=\"1.0\" encoding=\"utf-8\"?><checkstyle version=\"4.3\">";
```

Comment:
Not sure – I just run `grunt`, and get errors in this file. Wouldn't that use the just compiled version of TSLint?